src/test/eo/org/eolang/math/number-tests.eo

```diff
@@ -648,6 +648,77 @@
     .power 10
     $.equal-to 2655.992279142402
 
+[] > sqrt-check-zero-input
+  assert-that > @
+    abs.
+      number
+        minus.
+          0.0
+          sqrt.
+            number 0.0
+    $.less-than 0.00000000001
+
+[] > sqrt-check-negative-input
+  assert-that > @
+    is-nan.
+      sqrt.
+        number -0.1
+    $.equal-to TRUE
+
+[] > sqrt-check-float-input
+  assert-that > @
+    abs.
+      number
+        minus.
+          2.0
+          sqrt.
+            number
+              4.0
+    $.less-than 0.00000000001
+
+[] > sqrt-check-int-input
+  assert-that > @
+    abs.
+      number
+        minus.
+          9.0
+          sqrt.
+            number
+              81
+    $.less-than 0.00000000001
+
+[] > sqrt-check-nan-input
+  assert-that > @
+    is-nan.
+      sqrt.
+        number nan
+    $.equal-to TRUE
+
+[] > sqrt-check-max-int
+  9223372036854775807 > max-int!
+  assert-that > @
+    abs.
+      number
+        minus.
+          3037000499.97605
+          sqrt.
+            number
+              max-int
+    $.less-than 0.00001
+
+[] > sqrt-check-infinity-1
```

Comment:
@levBagryansky maybe it would be better to name it `sqrt-check-positive-infty`?

Reply:
@includealex I do not think this change plays role

Reply:
@levBagryansky please check this one https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

Reply:
@includealex What do you mean? Your suggestion is still compound. Please check how other tests are named in objectionary. For example https://github.com/objectionary/eo-strings/blob/master/src/test/eo/org/eolang/txt/text-tests.eo#L176. There are numerous of these cases. Your suggestion does not make the name clearer. I think then we must name it `sqrt-check-positive-infinity-in-input-positive-infinity-in-output`.  So I do not think it is a good decision

Reply:
@levBagryansky as it was written in the article, we should pay attention to description. Why do you stick to the idea that changing `...1` to `...positive` in the name of the test will not make the name clearer?

Reply:
I have explained it in the comment above. If you realy want to make it clearer name the test `sqrt-check-positive-infinity-in-input-positive-infinity-in-output`